src/config.json

```diff
@@ -382,6 +382,55 @@
 				}
 			]
 		},
+		{
+			"name": {
+				"en": "React compatibility",
+				"zh": "React 兼容性"
+			},
```

Comment:
I moved this section up as compatibility, for better or worse, is a major part of our ecosystem and draw. Most users are probably going to want to read that right after intro + essentials, debugging can get bumped down a slot.